scripts/deploy-airnode.ts

```diff
@@ -0,0 +1,22 @@
+import { join } from 'path';
+import { runAndHandleErrors, runShellCommand } from '../src';
+
+const main = async () => {
+
+  const airnodeDeploymentPath = join(__dirname, '../airnode-deployment');
+  const secretsFilePath = join(__dirname, '../airnode-deployment/aws.env');
+  const deployCommand = [
+    `docker run -it --rm`,
+    `-e USER_ID=$(id -u) -e GROUP_ID=$(id -g)`,
+    `--env-file ${secretsFilePath}`,
+    `-v ${airnodeDeploymentPath}:/app/config`,
+    `-v ${airnodeDeploymentPath}:/app/output`,
+    `api3/airnode-deployer:latest deploy --skip-version-check`,
```

Comment:
The `config.json` and `secrets.env` files are tied to a specific version (`0.3.1` at the moment) so I recommend setting that here to avoid issues once the `0.4` images are released.
```suggestion
    `api3/airnode-deployer:0.3.1 deploy --skip-version-check`,
```